applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -445,6 +465,7 @@ func mutatePod(admissionReview *admissionv1.AdmissionReview) (*admissionv1.Admis
 				}
 				// inject TPU_WORKER_ID
 				if getEnvironmentVariable("TPU_WORKER_ID", container) == "" {
+					klog.Infof("Injecting TPU_WORKER_ID %d to Pod", tpuWorkerID)
```

Comment:
Log something that would identify the pod (e.g. the host and replica numbers)

Reply:
Done, added the pod name and replica index.